notebooks/iatraoui.ipynb

```diff
@@ -192,8 +223,33 @@
     "                      0 where no polygon were found, and integers representing classes in order of occurence in the GeoDataFrame\n",
     "    \"\"\"\n",
     "\n",
-    "    ...  # TODO\n",
-    "    raise NotImplementedError\n"
+    "    mapping = {\n",
+    "        'WATER' : 1,\n",
+    "        'FOREST' : 2,\n",
+    "        'FARM' : 3\n",
+    "    }\n",
+    "\n",
+    "    _, x_shape, y_shape = data_array.shape\n",
```

Comment:
Here, the shape is _hardcoded_: the 3 dimensions are retrieved in a fixed order, meaning if these order was to be changed, the `_`, `x_shape` and `y_shape` variables would become out of order and incorrect

There is a more generic way: 

First, remove the dimension you are not interested in by selecting a slice. This corresponds to the `_` variable, so this is the `band` dimension:

```python
data_array.isel(band=0, drop=True)
```

`isel` is used (Dimension lookup by name, and Index lookup by integer), because we are not interested into the actual data, so any slice would work. We are interested by removing the `band` dimension, by name. Thanks to "Dimension lookup by name", the code becomes independant of the actual order of dimensions, and there is no need for unpacking `x` or `y`.
 
Finally you can have something like:

```python
shape = xda.isel(band=0, drop=True).shape
```